Source/com/drew/metadata/mp4/media/Mp4UserBoxDirectory.java

```diff
@@ -0,0 +1,37 @@
+package com.drew.metadata.mp4.media;
+
+import com.drew.lang.annotations.NotNull;
+
+import java.util.HashMap;
+
+public class Mp4UserBoxDirectory extends Mp4MediaDirectory {
+    public static final Integer TAG_UUID = 901;
+    public static final Integer TAG_LENGTH = 902;
```

Comment:
This tag is redundant, given it can be derived from the data itself.

Reply:
- Added XmpAtom for Mov format
- Fixed issue with offset in mp4 if size mdat box is 64-bit (it's typically only the Media Data Box)
- Added mp4 and mov files with "UUID" box and "XMP_" atom and tests

Reply:
Thanks for the update. Sorry I wasn't more clear about the test data. We can't add test media data to this repo without it becoming huge. There's a separate repo for this:

https://github.com/drewnoakes/metadata-extractor-images

I uploaded the test data you provided (thanks very much for preparing these):

https://github.com/drewnoakes/metadata-extractor-images/commit/0f1e3cd81cbbc5c0fba6956555c4e4fa8af9b175

I'm happy to edit your PR's commits to remove those files, unless you'd like to do so.

Reply:
yes, sure, you can move these files, thanks